common/statsd/statsd.cc

```diff
@@ -70,8 +70,10 @@ class StatsdClientWrapper {
     void timing(const CounterImpl::Timing &tim) { // type: ms
         auto nanoseconds = (tim.end.usec - tim.start.usec) * 1'000;
         if (tim.tags != nullptr) {
-            addMetric(absl::StrCat(tim.measure, ".duration_ns"), nanoseconds, "ms",
-                      *tim.tags); // format suggested by #observability (@sjung and @an)
+            // format suggested by #observability (@sjung and @an)
+            addMetric(absl::StrCat(tim.measure, ".duration_ns"), nanoseconds, "ms", *tim.tags);
+        } else {
+            addMetric(absl::StrCat(tim.measure, ".duration_ns"), nanoseconds, "ms", {});
```

Comment:
tim.tags used to be a `vector`, which would have been default initialized to the empty vector, and we would have always called `addMetric`.